### ISSUE TITLE
Feature Request - Allow to load specific custom feed by default

### ISSUE THREAD

**Issue by dnet890** [sugyan/tuisky#35]
Hi
on normal bsky.app  we can open homepage with our own custom feed. But, on tuisky it seems we need to pick specific feed before load the session. Is it possible to just run tuisky and we can run our specific feed by default?

**Comment by sugyan** [comment]
Indeed, currently the user must first select a feed in each column session.
I have been wanting to create a feature to save the last feed or screen that was opened and restore it the next time `tuisky` is launched, but have not implemented it yet.

Or, I thought of automatically selecting and displaying the top of the pinned feeds, but since `tuisky` does not have a sidebar or tabs, it is not possible to create a lead to the feed list, so I dare to let the user select the feed first.
